lib/webhooks/types.ts

```diff
@@ -115,3 +120,41 @@ export interface AddHandlersParams {
 export interface WebhookProcessParams extends AdapterArgs {
   rawBody: string;
 }
+
+export interface WebhookValidateParams extends AdapterArgs {
+  rawBody: string;
+}
```

Comment:
Any reason not to do this?
```suggestion
export interface WebhookValidateParams extends WebhookProcessParams {}
```